pgptool-gui/src/main/resources/pgptool-gui-messages.properties

```diff
@@ -182,21 +182,21 @@ encrypBackMany.encryptBackAll=Encrypt back all
 encrypBackMany.action=Encrypt back files
 encrypBackMany.operationInfo=Encrypt back {0} file(s) that were previously decrypted and now are being monitored
 encrypBackMany.warningMissingRecipients=Some files were initially encrypted for recipients that we cannot find anymore (their public keys are missing)
-encrypBackMany.willBeEncryptedForAllSameRecipients=Each file will be encrypted back for all same recipients it was recorded upon decryption
+encrypBackMany.willBeEncryptedForAllSameRecipients=Each file will be encrypted back for all the same recipients that it was recorded upon decryption
 term.isIgnoreMissingRecipientsWarning=Ignore missing recipients warning
 term.isEncryptOnlyChanged=Encrypt only changed files
 term.isDeleteUnencryptedAfter=Delete un-encrypted files
 error.noRecipientsFound=No recipients found for this file (keys are missing)
 error.someKeysAreMissing=Cannot find keys for all recipients 
 error.fileNotFound=File not found
 encrypBackMany.summary=Encryption result: {0} success, {1} warning(s), {2} failed
-exception.userRequestedCancelation=Operation canceled
+exception.userRequestedCancellation=Operation canceled
```

Comment:
I think this is the main line (193) one for this file, to sync up the name with the file "UserRequestedCancellationException.java"
The rest of my comments in this file are spelling and grammar fixes.